.github/workflows/python-package.yml

```
@@ -25,10 +25,10 @@ jobs:
       LAB_OBO_PUBLIC_CLIENT_ID: ${{ secrets.LAB_OBO_PUBLIC_CLIENT_ID }}
 
     # Derived from https://docs.github.com/en/actions/guides/building-and-testing-python#starting-with-the-python-workflow-template
-    runs-on: ubuntu-latest  # It switched to 22.04 shortly after 2022-Nov-8
+    runs-on: ubuntu-22.04  # ubuntu-latest switched to 22.04 shortly after 2022-Nov-8
```

https://github.com/actions/runner-images
https://github.blog/changelog/2024-09-25-actions-new-images-and-ubuntu-latest-changes/

```suggestion
    runs-on: ubuntu-22.04  # ubuntu-latest switched to 24.04 shortly after 2024-09-23
```

---

Good catch. That was an outdated comment that can actually be removed. Now updated.